housekeeper/store/filters/version_filters.py

```diff
@@ -9,20 +10,28 @@ def filter_version_by_id(versions: Query, version_id: int, **kwargs) -> Query:
     return versions.filter(Version.id == version_id)
 
 
+def filter_version_by_date(versions: Query, before_date: datetime, **kwargs) -> Query:
+    """Return version by date."""
+    return versions.filter(Version.created_at < before_date)
+
+
 class VersionFilters(Enum):
     """Define Version filter functions."""
     FILTER_BY_ID: Callable = filter_version_by_id
```

Comment:
entry_id?

Reply:
It does filter based on the property id on the model, so I think this is Ok.